pkg/injector/pod_patch.go

```diff
@@ -271,6 +279,33 @@ func profilingEnabled(annotations map[string]string) bool {
 	}
 }
 
+func getResourceLimits(annotations map[string]string) (*v1.ResourceRequirements, error) {
+	r := v1.ResourceRequirements{
+		Limits: v1.ResourceList{},
+	}
+	cpuLimit, ok := annotations[daprCPULimitKey]
+	if ok {
+		cpuLimit, err := resource.ParseQuantity(cpuLimit)
+		if err != nil {
+			return nil, fmt.Errorf("error parsing sidecar cpu limit: %s", err)
```

Comment:
If user puts invalid value, then what happens? will the injection for the app be failed? or will app be deployed without sidecar?

Reply:
It logs as warning and continues without constraints.